tests/bin/libc.h

```diff
@@ -145,3 +145,35 @@ uid_t getuid() {
     );
     return rax;
 }
+
+
+uid_t geteuid() {
+    uid_t rax;
+    asm(
```

Comment:
you forgot the clobber registers

Reply:
Woop! I missed this comment. I'll do this today.

Reply:
Done